src/commands.cpp

```diff
@@ -1204,7 +1211,88 @@ bool CommandPutNodes::procresult(Result r)
     }
 #endif
 
-    if (r.wasErrorOrOK())
+    if (r.hasJsonArray() || r.hasJsonObject())
+    {
+        client->sendkeyrewrites();
```

Comment:
Yes good point - because all node updates are handled only in the actionpacket code now, calls like this relating to those changes should be made there.  Our own client app doesn't need that done before it gets notified of the command completing so we can leave it entirely to the actionpacket system - which already calls applykeys() frequently, which calls that function.   I'll remove this instance of the call.